powersimdata/scenario/execute.py

```diff
@@ -342,14 +342,9 @@ def __init__(self, data_access, scenario_info, grid, ct):
         )
 
     def create_folder(self):
-        """Creates folder on server that will enclose simulation inputs.
-
-        :raises IOError: if folder cannot be created.
-        """
+        """Creates folder on server that will enclose simulation inputs."""
         print("--> Creating temporary folder on server for simulation inputs")
-        _, _, stderr = self._data_access.makedir(self.TMP_DIR)
-        if len(stderr.readlines()) != 0:
-            raise IOError("Failed to create %s on server" % self.TMP_DIR)
+        self._data_access.makedir(self.REL_TMP_DIR)
```

Comment:
This previously worked since one path was a prefix of the other one. Just changed so it's consistent with the assumption in `makedir` that we are using a relative path.

Reply:
do we need to keep the `TMP_DIR` attribute defined in the constructor?

Reply:
Good catch, it's not used anywhere now